frameworks/C++/lithium/benchmark_config.json

```diff
@@ -2,7 +2,7 @@
   "framework": "lithium",
   "tests": [
     {
-      "default": {
+      "mysql": {
```

Comment:
The first test needs to be called `default`. Can you swap this back, please.

Reply:
Fixed in my last commit.